Rakefile

```diff
@@ -36,12 +38,16 @@ Cucumber::Rake::Task.new(:cucumber) do |task|
 end
 
 task :travis do
+  rspec_opts << "--format documentation"
```

Comment:
Is there an actual reason for this change? or was it just for debugging?

Reply:
Debugging. Although realistically we might want to switch to `--format process` since `fuu` requires a raw TTY which is incompatible with Travis. Thoughts?

Reply:
I don't think it's necessary to switch in this PR